### PR TITLE
Write up a script to import investigator basics

### PR DESCRIPTION
**Pull request by calebthompson**
```ruby
require 'erb'
require 'active_support/core_ext'
require 'net/http'
require 'json'

investigators = JSON.parse(Net::HTTP.get(URI("https://arkhamdb.com/api/public/cards/"))).select { _1["type_code"] == "investigator" }
renderer = ERB.new(DATA.read)
investigators.each do |investigator|
  className = investigator["name"].gsub(/[^\w]/, '')
  varName = className.dup; varName[0] = varName[0].downcase
  file = "../ArkhamHorrorLCG/src/Arkham/Types/Investigator/Cards/#{className}.hs"
  next if File.exist? file
  File.write(file, renderer.result(binding))
end

__END__
{-# LANGUAGE UndecidableInstances #-}
module Arkham.Types.Investigator.Cards.<%= className %> where

import Arkham.Types.Classes
import Arkham.Types.Investigator.Attrs
import Arkham.Types.Investigator.Runner
import Arkham.Types.Message
import Arkham.Types.Stats
import Arkham.Types.Token
import Arkham.Types.Trait
import ClassyPrelude
import Data.Aeson

newtype <%= className %>I = <%= className %>I Attrs
  deriving newtype (Show, ToJSON, FromJSON)

<%= varName %> :: <%= className %>I
<%= varName %> = <%= className %>I $ baseAttrs
  "<%= investigator['code'] %>"
  <%= investigator['name'].inspect %>
  Stats
    { health = <%= investigator['health'] %>
    , sanity = <%= investigator['sanity'] %>
    , willpower = <%= investigator['skill_willpower'] %>
    , intellect = <%= investigator['skill_intellect'] %>
    , combat = <%= investigator['skill_combat'] %>
    , agility = <%= investigator['skill_agility'] %>
    }
  [<%= investigator['traits'].sub(".", ",")[0...-1] %>]

instance (InvestigatorRunner env) => RunMessage env <%= className %>I where
  runMessage msg i@(<%= className %>I attrs@Attrs {..}) = case msg of
    ResolveToken ElderSign iid _skillValue | iid == investigatorId -> pure i
    _ -> <%= className %>I <$> runMessage msg attrs
```

### CODE REVIEW

**Comment by halogenandtoast** [comment]
Can this prioritize the lower valued card code for duplicated investigators, that way we prioritize the "set" versions rather than the promos. We can then figure out the promos later.

**Comment by calebthompson** [comment]
It is done, updating the script (it's correct in the commit, not the PR)

File: Arkham.cabal [hunk 1]
```diff
@@ -48,9 +48,52 @@ library
...
       Arkham.Types.Investigator.Attrs
       Arkham.Types.Investigator.Cards.AgnesBaker
+      Arkham.Types.Investigator.Cards.AkachiOnyele
+      Arkham.Types.Investigator.Cards.AmandaSharpe
+      Arkham.Types.Investigator.Cards.AshcanPete
+      Arkham.Types.Investigator.Cards.CalvinWright
+      Arkham.Types.Investigator.Cards.CarolynFern
+      Arkham.Types.Investigator.Cards.CarolynFern
       Arkham.Types.Investigator.Cards.DaisyWalker
+      Arkham.Types.Investigator.Cards.DaisyWalker
```
Comment (halogenandtoast) [inline Arkham.cabal:58]: Duplicate

File: Arkham.cabal [hunk 2]
```diff
@@ -48,9 +48,52 @@ library
...
+      Arkham.Types.Investigator.Cards.MarieLambeau
+      Arkham.Types.Investigator.Cards.MarkHarrigan
+      Arkham.Types.Investigator.Cards.MinhThiPhan
+      Arkham.Types.Investigator.Cards.NathanielCho
+      Arkham.Types.Investigator.Cards.NormanWithers
+      Arkham.Types.Investigator.Cards.PatriceHathaway
+      Arkham.Types.Investigator.Cards.PrestonFairmont
+      Arkham.Types.Investigator.Cards.RexMurphy
+      Arkham.Types.Investigator.Cards.RitaYoung
+      Arkham.Types.Investigator.Cards.RolandBanks
```
Comment (halogenandtoast) [inline Arkham.cabal:83]: Duplicate

File: src/Arkham/Types/Investigator.hs [hunk 3]
```diff
@@ -14,9 +14,52 @@ import Arkham.Types.Card
...
 import Arkham.Types.Investigator.Attrs
 import Arkham.Types.Investigator.Cards.AgnesBaker
+import Arkham.Types.Investigator.Cards.AkachiOnyele
+import Arkham.Types.Investigator.Cards.AmandaSharpe
+import Arkham.Types.Investigator.Cards.AshcanPete
+import Arkham.Types.Investigator.Cards.CalvinWright
+import Arkham.Types.Investigator.Cards.CarolynFern
+import Arkham.Types.Investigator.Cards.CarolynFern
 import Arkham.Types.Investigator.Cards.DaisyWalker
+import Arkham.Types.Investigator.Cards.DaisyWalker
```
Comment (halogenandtoast) [inline src/Arkham/Types/Investigator.hs:24]: Duplicate

File: src/Arkham/Types/Investigator.hs [hunk 4]
```diff
@@ -14,9 +14,52 @@ import Arkham.Types.Card
...
+import Arkham.Types.Investigator.Cards.MarieLambeau
+import Arkham.Types.Investigator.Cards.MarkHarrigan
+import Arkham.Types.Investigator.Cards.MinhThiPhan
+import Arkham.Types.Investigator.Cards.NathanielCho
+import Arkham.Types.Investigator.Cards.NormanWithers
+import Arkham.Types.Investigator.Cards.PatriceHathaway
+import Arkham.Types.Investigator.Cards.PrestonFairmont
+import Arkham.Types.Investigator.Cards.RexMurphy
+import Arkham.Types.Investigator.Cards.RitaYoung
+import Arkham.Types.Investigator.Cards.RolandBanks
```
Comment (halogenandtoast) [inline src/Arkham/Types/Investigator.hs:49]: Duplicate

**Pull request merged**